test/test_api.py

```diff
@@ -638,6 +638,28 @@ def test_delete_folder(self, mocker):
         self.assertEqual("DELETE", get_method(mocker))
         self.assertTrue(get_uri(args).endswith('/folders/' + UNIQUE_TEST_FOLDER))
 
+    @patch('urllib3.request.RequestMethods.request')
+    @unittest.skipUnless(cloudinary.config().api_secret, "requires api_key/api_secret")
+    def test_root_folders_allows_next_cursor_and_max_results_parameter(self, mocker):
+        """ should allow next_cursor parameter """
+        mocker.return_value = MOCK_RESPONSE
+        api.root_folders(next_cursor="foo", max_results=10)
```

Comment:
We prefer to use test values that represent close to the real world values. Please replace "foo" with some value that can be a valid next_cursor, you can take one from the PHP SDK